# 평균검정 
1. 단일표본검정
2. 두 독립표본의 검정
3. 짝지어진 표본의 검정  

## 단일표본 평균 검정(One sample T-test)
25마리 게의 체온을 측정한 데이터가 아래와 같을 때, 게의 평균 체온이 24.3도 라고 할 수 있는지 검정해보자
- 귀무가설 : 게의 평균 체온은 24.3도이다.
- 대립가설 : 게의 평균 체온은 24.3도가 아니다.

In [1]:
crab = [25.8, 24.6, 26.1, 22.9, 25.1, 27.3, 24.0, 24.5, 23.9, 26.2,
       24.3, 24.6, 23.3, 25.5, 28.1, 24.8, 23.5, 26.3, 25.4, 25.5,
       23.9, 27.0, 24.8, 22.9, 25.4]

In [2]:
import pandas as pd

In [4]:
crab_df = pd.DataFrame(crab)

crab_df.head()

0
0  25.8
1  24.6
2  26.1
3  22.9
4  25.1

In [5]:
from scipy import stats
crab_test = stats.ttest_1samp(crab, 24.3)
print("T-value = %.3f, p-value = %.3f" % crab_test)

T-value = 2.713, p-value = 0.012


p-value가 0.012로 0.05보다 작기 떄문에 귀무가설을 기각합니다.  
즉, 게의 평균 체온이 24.3도라고 할 수 없습니다.